modules/sc-mesh-secure-deployment/src/server-mesh.py

```diff
@@ -65,9 +70,11 @@ def add_message(uuid):
         aux = aux_ubuntu if uuid == 'Ubuntu' else aux_openwrt
         if ip_mesh == IP_PREFIX + '.2':  # First node, then gateway
             aux['gateway'] = True
-            add_default_route(ip_address)  # we will need to add the default route to communicate
+            add_default_route(ip_mesh, ip_address)  # we will need to add the default route to communicate
         else:
             aux['gateway'] = False
+        if ip_mesh == IP_PREFIX + '.3':  # TODO: find smart way to set this value. Currently: only one server == '.3'
```

Comment:
A simple solution could be based on # of nodes joined who AREN'T servers. i.e. keep track of # of nodes joined then mod it so every 5th node (for example) is delegated as a server. 

Additionally, we need to think about scalability. There should probably be a max # of servers per mesh.